source/server/configuration_impl.cc

```diff
@@ -106,12 +106,7 @@ void MainImpl::initializeTracers(const envoy::config::trace::v2::Tracing& config
 
   // Now see if there is a factory that will accept the config.
   auto& factory = Config::Utility::getAndCheckFactory<HttpTracerFactory>(type);
-  if (factory.requiresClusterName() && server.localInfo().clusterName().empty()) {
-    throw EnvoyException(fmt::format("cluster name must be defined for the tracing driver {}. See "
-                                     "--service-cluster option.",
-                                     type));
-  }
-  http_tracer_ = factory.createHttpTracer(*driver_config, server, *cluster_manager_);
```

Comment:
This is a code change, but it removes a redundant param which is not needed. Cluster manager is contained in the server.